marklogic-data-hub-central/ui/e2e/cypress.config.js

```diff
@@ -15,7 +16,7 @@ module.exports = defineConfig({
     toConsole: true,
   },
   retries: {
-    runMode: 1,
```

Comment:
@vmendez-marklogic I have observed few tests failing at first attempt and then going fine in retry and progress further. So may be we can leave the retries as 1 for now. Rest all changes look good to me.

Reply:
Okay! I will revert that change, and we can discuss about retryability, and create a task if it is needed